samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+          await Excel.run(async (context) => {
+            context.application.cultureInfo.datetimeFormat.load([
+              "longDatePattern",
+              "shortDatePattern",
+              "dateSeparator",
+              "longTimePattern",
+              "timeSeparator"
+            ]);
+            await context.sync();
+
+            // System settings are read-only and set in the user's operating system.
+            const systemLongDatePattern = context.application.cultureInfo.datetimeFormat.longDatePattern;
+            const systemShortDatePattern = context.application.cultureInfo.datetimeFormat.shortDatePattern;
+            const systemDateSeparator = context.application.cultureInfo.datetimeFormat.dateSeparator;
+            const systemLongTimePattern = context.application.cultureInfo.datetimeFormat.longTimePattern;
+            const systemTimeSeparator = context.application.cultureInfo.datetimeFormat.timeSeparator;
+
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const longDate = sheet.getRange("A2:B2");
+            longDate.values = [["Long date", systemLongDatePattern]];
+            const shortDate = sheet.getRange("A3:B3");
+            shortDate.values = [["Short date", systemShortDatePattern]];
+            const shortDate = sheet.getRange("A4:B4");
+            shortDate.values = [["Date separator", systemDateSeparator]];
+            const shortDate = sheet.getRange("A5:B5");
+            shortDate.values = [["Long time format", systemLongTimePattern]];
+            const shortDate = sheet.getRange("A6:B6");
+            shortDate.values = [["Time separator", systemTimeSeparator]];
+
+            sheet.tables
+              .getItemAt(0)
+              .getRange()
+              .format.autofitColumns();
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        -| <section class="ms-font-m"> <p>This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.</p> </section>
+        <section class="samples ms-font-m"> <h3>Setup</h3> <button id="setup" class="ms-Button"> <span class="ms-Button-label">Setup</span> </button> </section>
+        <section class="samples ms-font-m"> <h3>Try it out</h3> <button id="display-date-time-setting" class="ms-Button"> <span class="ms-Button-label">Display date/time settings</span> </button> <p></p> <button id="write-date-time-setting" class="ms-Button"> <span class="ms-Button-label">Write date/time settings</span> </button> </section>
```

Comment:
Can we spread this out into more lines? You might want to copy-and-paste this section from Script Lab, since the export feature doesn't handle the HTML section well.

Reply:
Something happened with the formatting -- I explicitly followed your directions about copy/pasting from the HTML panel, but somehow this is all still minified, even in my local version. I wonder if it happened when I ran yarn start? Or maybe something with my VSCode... any ideas? It also added the extra -| on both files.

Reply:
It might be something happening during yarn start. We can take a deeper look or fix it in a subsequent PR.